liferay-lima/liferay-portal
# LPS-166870 Clean up templates view

# What is this about?

This pr is a clean up of the templates jsp. No logic changes.

# How do I test this?

Go to the templates tab. If you:
- Can create templates,
- See the templates listed in the search container,
- Search for templates, and
- Delete templates, both individually and in bulk,

then everything works as expected.
